src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/StatusHistoryDto.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.refunds.dtos.responses;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.*;
+
+import java.sql.Timestamp;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@Builder(builderMethodName = "buildStatusHistoryDtoWith")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+public class StatusHistoryDto implements Comparable<StatusHistoryDto> {
```

Comment:
As discussed can you please use Comparator.Comparing  of streams for sorting based on date created?

Reply:
Done